### ISSUE TITLE
Problem: crontab uses timex, quantum used calendar

### ISSUE THREAD

**Issue by michalmuskala** [c-rack/quantum-elixir#176]
Since the dependency of quantum - `crontab` already uses `timex`, maybe quantum itself should consider using this. This would ultimately allow skipping one dependency overall.

**Comment by maennchen** [comment]
That should be easy to do. It's just in very few places.

```
lib/quantum/executor.ex:15:      :utc   -> dt |> Calendar.DateTime.from_erl!("Etc/UTC") |> Calendar.DateTime.shift_zone!(tz_final)
lib/quantum/executor.ex:17:      _      -> dt |> Calendar.DateTime.from_erl!(tz_final)
lib/quantum/timer.ex:25:    |> Calendar.DateTime.now!
lib/quantum/timer.ex:26:    |> Calendar.DateTime.to_erl
```

**Comment by benwilson512** [comment]
Even better, can we try to make it configurable? I guess that's more of an issue for `crontab` but for those of us on Calendar, the inclusion of timex is an unfortunate addition.

**Comment by maennchen** [comment]
@benwilson512 That's where the PR is heading.

**Comment by maennchen** [comment]
I need some help for the calendar implementations though. Is someone interested to provide the adapter module for calendar in quantum and crontab?

**Comment by maennchen** [comment]
Closed with #177
